examples/api-samples/src/browser/menu/sample-menu-contribution.ts

```diff
@@ -50,6 +54,10 @@ const SampleQuickInputCommand: Command = {
     category: 'Quick Input',
     label: 'Test Positive Integer'
 };
+const SampleReactDialog: Command = {
+    id: 'sample-command-react-dialog',
+    label: 'Sample React Dialog'
+};
```

Comment:
Minor: Since this is specifically to test the select component, we should give this a better name
```suggestion
const SampleSelectDialog: Command = {
    id: 'sample-command-select-dialog',
    label: 'Sample Select Component Dialog'
};
```